docs/scripts/macros.njk

```diff
@@ -9,6 +22,7 @@
 <p/>
 {% endmacro %}
 
+
```

Comment:
Not sure about this but are two blank lines required?

Reply:
I usually use two blank lines between macros, similar to python convention of putting two blank lines between functions.